app/pages/blog/index.vue

```
@@ -1,24 +1,70 @@
+const hasPrevPage = computed(() => currentPage.value > 1)
+
+// SEO for paginated pages
 const seoImage = 'https://images.pexels.com/photos/27277185/pexels-photo-27277185.jpeg'
-useLangMeta('Blog.meta', seoImage)
+const pageTitle = computed(() => {
+  if (currentPage.value > 1) {
+    return `Blog - Page ${currentPage.value}`
+  }
+  return 'Blog'
+})
+useLangMeta('Blog.meta', seoImage, 'prefix', undefined, pageTitle.value)
```

ESLint

[error] 'useLangMeta' is not defined.

(no-undef)